### PR TITLE
Add my-collections page

### PR DESCRIPTION
**Pull request by Lbqds**

### CODE REVIEW

**Comment by h0ngcha0** [comment]
Question: in the `My Collection` page, is the intention to list all collection I created, or collection of the NFTs that I currently own?

File: pages/my-collections.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import { useAlephiumConnectContext } from '@alephium/web3-react';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import Link from 'next/link';
+import { motion } from 'framer-motion';
+import { shortenName } from '../utils';
+
+interface NFTCollection {
+  _id: string  // NFTCollection contract id
+  type: 'NFTOpenCollection' | 'NFTPublicSaleCollection'
```
Comment (h0ngcha0) [inline pages/my-collections.tsx:15]: Seems that we are still setting `NFTPublicSaleCollectionSequential` in the code.

File: pages/my-collections.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            className="flex justify-center items-center hover:scale-110 transition-all duration-500"
+            src={collection.image}
+            layout="fill"
+            objectFit="cover"
+            alt={`nftCollection${collection._id}`}
+          />
+        </div>
+        <div className="mt-3 flex flex-col">
+          <p className="font-poppins dark:text-white text-nft-black-1 font-semibold text-xs minlg:text-xl">
+            {collection.name.length > 14 ? shortenName(collection.name) : collection.name}
```
Comment (h0ngcha0) [inline pages/my-collections.tsx:41]: [Nit] Perhaps we can show the `totalSuppy` in the card? Similar to extension wallet.

**Comment by Lbqds** [comment]
> Question: in the `My Collection` page, is the intention to list all collection I created, or collection of the NFTs that I currently own, or both?

Only I created collections for now, but we can also support the second one.

File: pages/my-collections.tsx (hunk 1 above)
Comment (Lbqds) [reply pages/my-collections.tsx:15]: I have cherry-picked it from the old branch, will fix it, thanks!

**Comment by h0ngcha0** [comment]
> > Question: in the `My Collection` page, is the intention to list all collection I created, or collection of the NFTs that I currently own, or both?
> 
> Only I created collections for now, but we can also support the second one.

That would be great. Since if I bought an NFT, I would probably expect to see it in my collections. 

I am actually a bit unclear about if we click on the collection in my collection, shall we see all NFTs within that collection or only my NFTs in that collection. WDTY? 

Another question is if we have my collection tab, do we still need my nft tab?

**Comment by Lbqds** [comment]
> I am actually a bit unclear about if we click on the collection in my collection, shall we see all NFTs within that collection or only my NFTs in that collection. WDTY?

Currently we can see all NFTs because it just redirects to the `collection-details` page, perhaps we can refer to Opensea's approach. I will check it later.

> Another question is if we have my collection tab, do we still need my nft tab?

I was hesitant before, but I think users might want to see my nfts directly, so I kept it for now.

**Comment by h0ngcha0** [comment]
> > I am actually a bit unclear about if we click on the collection in my collection, shall we see all NFTs within that collection or only my NFTs in that collection. WDTY?
> 
> Currently we can see all NFTs because it just redirects to the `collection-details` page, perhaps we can refer to Opensea's approach. I will check it later.
> 
> > Another question is if we have my collection tab, do we still need my nft tab?
> 
> I was hesitant before, but I think users might want to see my nfts directly, so I kept it for now.

If feels like if we only show user's own NFT when clicking the collection in My Collection tab, then we might not need the My NFT page, since they are basically two views of the same info.

**Comment by h0ngcha0** [comment]
Close since everything is in https://github.com/alephium/alephium-nft/pull/28 now

**Pull request closed without merging**